app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js

```diff
@@ -48,25 +60,61 @@ const selectedPrice = {
   price: '1.64',
 };
 
-const mocks = () => {
+const selectedPriceWithoutData = {
+  priceId: 2,
+  provisioningType: 'Patient',
+  type: 'flat',
+  currencyCode: 'GBP',
+  itemUnit: {
+    name: 'patient',
+    description: 'per patient',
+  },
+  timeUnit: {
+    name: 'year',
+    description: 'per year',
+  },
+};
```

Comment:
mmm I don't think using this as the returned data makes this an accurate ui test. Populating the form with blank data and then clicking submit.  BAPI should most likely return these values and the user will then delete the contents of these from the fields and then click submit. I think commencement date ui test is an example of where we delete data in the form and click submit

Reply:
but then the end result is the same that nothin is populated in the box before the submit button is clicked?

Reply:
I totally agree but I don't think is the right way to mimic that. We should reflect how that app is going behave and the users interactions with it. I think it's just the price that is populated and can be removed before submitting. 

This way if someone else comes to look at these we know we are always expecting the price to be provided to us and it's possible for this value to be deleted by the user and submitted

Reply:
yeah that's fair will delete the price instead then